### ISSUE TITLE
Display Scaling for 1080p Laptop Screens

### ISSUE THREAD

**Issue by CountryfiedLinux** [mate-desktop/mate-desktop#403]
#### Expected behaviour
To be able to scale my 1080p laptop screen to 125% or something close to that.

#### Actual behaviour

Fractional display scaling is not present in the display settings.
#### Steps to reproduce the behaviour
Go to display settings and look for scaling. Fractional settings are not available.

#### MATE general version

1.20
#### Linux Distribution
Ubuntu Mate 19.04

**Comment by vkareh** [comment]
You can also change the font DPI

**Comment by CountryfiedLinux** [comment]
I realize I can change font DPI, that's not an issue. Changing font DPI doesn't change the desktop, just the font. The desktop aspects are still tiny on 1080p laptop screens.

**Comment by vkareh** [comment]
True. Once we can support Wayland instead of X11 we should be able to provide fractional scaling. Marking as feature request for now.

**Comment by CountryfiedLinux** [comment]
Ok thanks!

**Comment by shirishag75** [comment]
> True. Once we can support Wayland instead of X11 we should be able to provide fractional scaling. Marking as feature request for now.

Any update as to the state of things as far as Wayland is concerned as of date ?

**Comment by adriesse** [comment]
I hope this hasn't been forgotten!

**Comment by lukefromdc** [comment]
Nothing much can be done about this so long as we are on x11, a full MATE session on Wayland with icons on the desktop(requires a new desktop canvas from scratch) , all the panel applets (must be ported to in-process), and any bad bugs squashed is a ways out.
